libs/interpret/src/lib/MLIDashboard/ValidateProperties.ts

```diff
@@ -155,7 +155,7 @@ export class ValidateProperties {
         !this.validate2D(
           globalExp,
           [this.featureLength, this.classLength],
-          localization.Interpret.ValidationErrors.localFeatureImportance
+          localization.Interpret.ValidationErrors.globalFeatureImportance
         )
```

Comment:
not sure I understand this, but isn't the function validating local importance? Then change the error from local to global?

Reply:
it is validation global(line 156)
but it was using error message for local importance.
So I correct it by using global message